server/src/db/migrations/20210722191748_createVotes.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("votes", (table) => {
+    table.bigIncrements("id")
+    table.string("direction").notNullable()
```

Comment:
Im okay with this being a string, just know that this can allow for more errors when evaluating the type of vote.

another option includes limitng a vote's direction to being -1, 0 and 1. -1 and 1 being downvote and upvote, and 0 tracking that a specific user has potentially "unvoted". You might circle back around to this when allowing users to vote

Reply:
we spoke through the implementation. the idea is for the "votes" table to track entries of votes. if a user/admin decides to remove their vote, we will delete that vote entry from the "votes" table.